allen-cell-animated/timelapse-colorizer
# fix: Bad feature values are now shown as NaN instead of Infinity

Problem
=======
Closes #329, a bug where `NaN` values are shown as `Infinity`.

*Estimated review size: tiny, 5 minutes*

Solution
========
- Added a comment explaining why `Infinity` values are stored internally.
- Added a check that replaces `Infinity` values with `NaN` on the hover tooltip.
- Deleted unused shader.
- Minor formatting changes to shader code.

## Type of change
Please delete options that are not relevant.

* Bug fix (non-breaking change which fixes an issue)

Steps to Verify:
----------------
1. Open the preview link and hover over the cells. Greyed-out cells should appear as `NaN` instead of `Infinity`: https://allen-cell-animated.github.io/timelapse-colorizer/pr-preview/pr-377/viewer?collection=https%3A%2F%2Fdev-aics-dtp-001.int.allencell.org%2Fdan-data%2Fcolorizer%2Fdata%2Fnucmorph-apr2024%2Fcollection.json&dataset=Mama+Bear&feature=time&t=396
2. Compare with main build: https://allen-cell-animated.github.io/timelapse-colorizer/viewer?collection=https%3A%2F%2Fdev-aics-dtp-001.int.allencell.org%2Fdan-data%2Fcolorizer%2Fdata%2Fnucmorph-apr2024%2Fcollection.json&dataset=Mama+Bear&feature=time&t=396
3. 
![image](https://github.com/allen-cell-animated/timelapse-colorizer/assets/30200665/cb207805-842a-4a7d-baf0-e42436cffadc)
